VERSION

```diff
@@ -0,0 +1 @@
+v2.3.0
```

Comment:
what's this for?

Reply:
this is used for build-package.yml which used "cat ./version"